src/osdc/ObjectCacher.cc

```diff
@@ -2603,12 +2605,10 @@ void ObjectCacher::verify_stats() const
 
   loff_t clean = 0, zero = 0, dirty = 0, rx = 0, tx = 0, missing = 0,
     error = 0;
-  for (vector<ceph::unordered_map<sobject_t, Object*> >::const_iterator i
-	 = objects.begin();
+  for (auto i = objects.begin();
```

Comment:
This was `const_iterator` in the existing code, need to switch to `cbegin()`/`cend()`?